### PR TITLE
get/api/topics

### PR DESCRIPTION
**Pull request by Topecoding206**
this commit add functionality to get topics

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
+  });
+});
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:27]: Please add a test for status 404

File: errorHandler.js [hunk 2]
```diff
@@ -0,0 +1 @@
+exports.handleError400 = (error, request, response, next) => {};
```
Comment (Beth-McCormack) [inline errorHandler.js:1]: Please include handlers for status 404 and 500 here instead of 400

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

File: errorHandler.js (hunk 2 above)
Comment (Topecoding206) [reply errorHandler.js:1]: ok

**Pull request merged**